admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
+| `/participants/{name}/endpoints`  | View participant endpoints  | Not supported  | Add/Update participant endpoints  | Not supported  |
+| `/participants/{name}/limits`  | View participant limits  |  Adjust participant limits | Not supported  |  Not supported |
+| `/participants/{name}/positions`  | View participant positions  | Not supported  | Not supported  | Not supported  |
+| `/participants/{name}/accounts`  | View participant accounts and balances  |  Not supported | Create Hub accounts  |  Not supported |
+| `/participants/{name}/accounts/{id}`  |  Not supported | Update participant accounts  | Record Funds In or Out of participant account  | Not supported  |
+| `/participants/{name}/accounts/{id}/transfers/{transferId}`  | Not supported  | Not supported  | Record a Transfer as a Funds In or Out transaction for a participant account  |  Not supported |
+| `/participants/{name}/initialPositionAndLimits`  | Not supported  | Not supported  |  Add initial participant limits and position | Not supported  |
+| **`/settlementModels`**  | View all settlement models  | Not supported  | Create a settlement model  |  Not supported |
+| `/settlementModels/{name}`  | View settlement model by name  | Update a settlement model (activate/deactivate a settlement model) | Not supported  |  Not supported |
+| **`/transactions/{id}`**  | Retrieve transaction details by `transferId`  | Not supported  | Not supported  | Not supported |
+
+## API Resource `/participants`
+
+The services provided by the resource `/participants` are primarily used by the Hub Operator for viewing, creating, and updating participant-related details, such as limit (Net Debit Cap), position, or endpoints configured.
+
+### `GET /participants`
+
+The `GET /participants` request is used to retrieve information about all participants.
+
+#### Data model
+
+The `GET /participants` request sends an empty request body.
+
+The following data is returned for each participant in the case of a successful operation:
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `name`  |  yes | [String(2..30)](#string) | The name of the participant. |
+| `id`  |  yes | [String](#string) | The identifier of the participant in the form of a fully qualified domain name combined with the participant's `fspId`. |
+| `created`  |  yes | [DateTime](#datetime)  | Date and time when the participant was created. |
+| `isActive`  |  yes | [Integer(1)](#integer) | A flag to indicate whether or not the participant is active. Possible values are `1` and `0`. |
+| `links`  |  yes | [Self](#self) | List of links for a Hypermedia-Driven RESTful Web Service. | 	
+| `accounts`  |  yes | [Accounts](#accounts) | The list of ledger accounts configured for the participant. |
+
+### `POST /participants`
+
+The `POST /participants` request is used to create a participant in the Hub.

```

Comment:
nit: cut down on redundant language. We are already in the `POST /participants` section, and we already know it's a  HTTP request
```suggestion
Creates a participant in the Hub.

```

Reply:
OK, agree.